### ISSUE TITLE
No easy way to export-import Evernote notes on Linux

### ISSUE THREAD

**Issue by michaldybczak** [laurent22/joplin#4647]
On Linux I have two alternatives:
Tusk - I can import notes as html or markdown one by one, impossible when having a hundreds of notes
Nixnote - exports to nnex file which cannot be used in Joplin, despite what your FAQ says. The thread on Joplin forums is not resolved.

I also tried to install Windows desktop app through wine but... there is no way to get windows desktop app. There is simply no installation file. I'm constantly directed to Windows Store and there I can't install anything, because I'm on Linux.

Circle is closed. This is frustrating.

Without a way out I have no way out :(.

EDIT: I found out that there is an Evernote Beta app for Linux. I was able to install it from AUR. I was able to import notes but... the app has limitation 50 marked notes so I had to do it piece by piece and import it the same way. The biggest issue here is thou that after import, all notes are with the default import notebooks. The original notebooks structure is lost and notes are loose. This is a horror :(. Did I do something wrong? How can I import notes with the notebooks?

EDIT2: I found out I can export whole notebooks so I can recreate the structure, but it will be a tedious task.

What we need is:

1. A way to import .nnext Nixnote files or:
2. A good FAQ showing how to get the Evernote Beta on various distributions and then how to import notebooks with notes, eventually show how to import all notebooks at once if it's possible.

**Comment by CalebJohn** [comment]
I feel for you, when I initially transferred everything to Joplin I had to borrow my girlfriends computer, install Evernote and export that way. 

That said, this really is a losing battle. Evernote seems to routinely change the enex export format, requiring trial and error patching to get imports working correctly. That and the Evernote export format is lossy (as you've already seen with notes losing all notebook information). These are problems that can not be fixed on the Joplin side.

> 1. A way to import .nnext Nixnote files or:

~~This would indeed be nice, the nixnote format is presumably more stable. ~~
I just checked and it looks like Nixnote has not had any updates since 2017, so indeed the export format must be quite stable :). With the way that evernote is developed I'm surprised that Nixnote continues to work at all. In either case, Joplin supports custom importers through [the plugin interface](https://joplinapp.org/api/references/plugin_api/classes/joplininterop.html). This might make a good project for someone thats interested.

> 2. A good FAQ showing how to get the Evernote Beta on various distributions and then how to import notebooks with notes, eventually show how to import all notebooks at once if it's possible.

Would you mind writing something up? We could post it to a forum topic and link to it from the README. The Evernote export situation is always changing so it make sense to include the information in something a bit more fluid like a forum topic.

**Comment by michaldybczak** [comment]
I took the effort and exported and then imported notebooks one by one. I had ca. 2,5 dozens of them, but it was still doable. This way I preserved the structure on joplin. So the process is not ideal and without that beta app, almost impossible. Anyway, first hardship is beaten.

Jooplin looks great and it was easy to add sync by dropbox. So far I'm very impressed (except that export/import thing ;) ).

I saw that there is a web clipper, this is my next stop. One of the reasons to abandon Evernote was its clipper that conflicted with my bank account site... which doesn't load when Evernote clipper is enabled.

I also wonder, how I will be able to add and modify notes on computers without joplin, like at work? Hopefully it will be possible through dropbox somehow. I don't plan it using extensively at work, so small edits in sync file should be fine. Having clipper is more important, because I was to save some important documents.

I really hope I can move fully to Joplin and leave Evernote behind. Testing begins :).

As to writing something on forum, this is possible, but the problem is time. It's 1 am and it's time to go to sleep. I wake up at 6 am... and I work ca. 10 hours at least every day. I like to be active on open source related forums (Manjaro, since I'm using it) but time is limited so I'm not to active lately. I can't make any promises but if I like Joplin, I may write something :).

**Comment by michaldybczak** [comment]
Since I managed to import all my notes, it's not an issue for me anymore. However from the bigger perspective, the issue still exists - there is no way to easy import notes and notebooks from Evernote. It is doable but requires a time-consuming and not easy research, then testing, then frustrating realization that some ways don't work as promised, etc.

So here are possible solutions for this issue to be closed:

1. Joplin promises to work with  .nnext Nixnote files, but it doesn't. At least the info should be replaced, so it would no longer create confusion. Ideally, .nnext files should be acceptable and readable, if they are a consistent format.
2. A good FAQ showing how to get the Evernote Beta on various distributions and then how to import notebooks with notes, eventually show how to import all notebooks at once if it's possible. - this was the way that worked for me, although it was very hard to find it and in the end I had to import notebooks one by one, which took time and I had few dozens of them. Still, I did the job, and now I fully moved to Joplin. This wouldn't be possible if not Evernote Beta, an app that is nowhere mentioned in Joplin documentation.
